### PR TITLE
fix: remove logic that sets bpn before validation

### PR DESCRIPTION
**Pull request by JordanGerada3**
## Description

- Removed prop from input field that was setting the BPNL on the form without being firstly validated

## Why

When a user would insert an invalid BPNL in the autofill field, the BPNL was updating in the form despite being invalid, leading to the possibility of the user submitting the application with an incorrect BPNL. 

## Issue

#252 

## Checklist

- [x] I have followed the [contributing guidelines](https://github.com/eclipse-tractusx/portal-assets/blob/main/docs/developer/Technical%20Documentation/Dev%20Process/How%20to%20contribute.md#commit-and-pr-guidelines)
- [x] I have performed a self-review of my own code
- [x] I have successfully tested my changes locally
- [x] I have checked that new and existing tests pass locally with my changes

### CODE REVIEW

**Review by typecastcloud — APPROVED** [review]
Looks good.

**Comment by JordanGerada3** [comment]
> Looks good. Unintended behavior by the copy paste mess I made when adding onBlur logic. Thanks for the find!
> 
> Please update DEPENDENCIES file. See artifact here: https://github.com/eclipse-tractusx/portal-frontend-registration/actions/runs/10845856689

DEPENDENCIES updated!

**Review by shubhamv-ss — APPROVED** [review]
lgtm

File: CHANGELOG.md [hunk 1]
```diff
@@ -13,6 +13,7 @@
 - fixed country value not updating if user moves back and forth between steps [#232](https://github.com/eclipse-tractusx/portal-frontend-registration/pull/232)
 - removed code that always displays error message on screen [#230](https://github.com/eclipse-tractusx/portal-frontend-registration/pull/230)
 - updated the logic for maxFiles count in upload docs step [#234](https://github.com/eclipse-tractusx/portal-frontend-registration/pull/234)
+- remove logic that sets bpn before validation [#253](https://github.com/eclipse-tractusx/portal-frontend-registration/pull/253)
```
Comment (evegufy) [inline CHANGELOG.md:16]: please move to unreleased section

Comment (JordanGerada3) [reply CHANGELOG.md]: @evegufy I've resolved this now.

**Review by evegufy — APPROVED** [review]
lgtm, thank you @JordanGerada3

**Review by evegufy — CHANGES_REQUESTED** [review]
sorry, didn't see it earlier, could you please still update the DEPENDENCIES file again?

**Comment by JordanGerada3** [comment]
> sorry, didn't see it earlier, could you please still update the DEPENDENCIES file again?

@evegufy it seems that the failure happened before the artefact generation, therefore I have nothing to update the dependencies with. I also haven't introduced any new dependencies in my PR. Do you know how to solve this?

**Comment by evegufy** [comment]
@JordanGerada3 sorry for the wait. We'll update the dependencies with another PR as this change doesn't include update to dependencies.

**Review by evegufy — APPROVED** [review]

**Pull request merged**